examples/model-mysql.js

```diff
@@ -86,8 +87,9 @@ module.exports.uploadPicture = async (reqFile, title) => {
   const dbRecord = {
     filename: newFilename,
     title,
+    author,
   };
 
   const [rows] = await sql.query(sql.format('INSERT INTO picture SET ?', dbRecord));
-  return { id: rows.insertId, title: dbRecord.title, file: config.webimg + dbRecord.filename };
+  return { id: rows.insertId, title: dbRecord.title, file: config.webimg + dbRecord.filename, author: dbRecord.author};
```

Comment:
please run eslint, or even better install `linter` and `linter-eslint` packages in Atom (or equivalents in whatever other editor you use) please and make sure the code lints without warnings/errors.